library/core/src/macros/mod.rs

```diff
@@ -1714,6 +1714,18 @@ pub(crate) mod builtin {
         /* compiler built-in */
     }
 
+    #[cfg(not(bootstrap))]
+    /// Unstable placeholder for type ascription.
```

Comment:
Is this right?

Reply:
nah lol i am just bad at copying and pasting